webapp/server.js

```diff
@@ -58,17 +57,41 @@ let useHttps = process.env.USE_HTTPS || config.USE_HTTPS;
 
 useHttps = useHttps.toLowerCase();
 
+// Production session data
+const session = require('express-session');
+var redis = require("redis");
+var redisStore = connect_redis(session);
+var redisClient = redis.createClient({
+    host: process.env.REDIS_HOST,
+    port: 6379,
+});
+
+const isSecure = env === 'production' && useHttps === 'true';
+if (isSecure) {
+  app.use(forceHttps);
+}
+
+// Support for parsing data in POSTs
+app.use(bodyParser.json());
+app.use(
+  bodyParser.urlencoded({
+    extended: true,
+  })
+);
+
+app.set('trust proxy', 1); // needed for secure cookies on heroku
```

Comment:
Are we running on heroku?

Reply:
Nope. That's left over from when it did.